[2022-06-12 16:52:16,917] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ingest_data.save_data_task scheduled__2022-06-12T02:24:00+00:00 [queued]>
[2022-06-12 16:52:16,959] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ingest_data.save_data_task scheduled__2022-06-12T02:24:00+00:00 [queued]>
[2022-06-12 16:52:16,960] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-12 16:52:16,961] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-12 16:52:16,962] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-12 16:52:17,021] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): save_data_task> on 2022-06-12 02:24:00+00:00
[2022-06-12 16:52:17,026] {standard_task_runner.py:52} INFO - Started process 5906 to run task
[2022-06-12 16:52:17,033] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'ingest_data', 'save_data_task', 'scheduled__2022-06-12T02:24:00+00:00', '--job-id', '684', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_dag.py', '--cfg-path', '/tmp/tmp360dt0_l', '--error-file', '/tmp/tmp254j9t8m']
[2022-06-12 16:52:17,035] {standard_task_runner.py:80} INFO - Job 684: Subtask save_data_task
[2022-06-12 16:52:17,176] {task_command.py:370} INFO - Running <TaskInstance: ingest_data.save_data_task scheduled__2022-06-12T02:24:00+00:00 [running]> on host DESKTOP-GDPK8BL.localdomain
[2022-06-12 16:52:17,670] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=save_data_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-12T02:24:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-12T02:24:00+00:00
[2022-06-12 16:52:17,686] {python.py:173} INFO - Done. Returned value was: None
[2022-06-12 16:52:17,715] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=save_data_task, execution_date=20220612T022400, start_date=20220612T145216, end_date=20220612T145217
[2022-06-12 16:52:17,811] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-12 16:52:18,013] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
